.github/workflows/continous-integration.yml

```diff
@@ -165,6 +165,10 @@ jobs:
       run: |
         make actions &
 
+    - name: Run duckling server
```

Comment:
I would put this in Makefile as a new target `run-duckling` and then modify `run` target as follows:
```
run: run-duckling
	poetry run rasa inspect --debug
```

Reply:
sure, can do.

Reply:
although I'd have to amend all make test targets too.

Reply:
Addressed: https://github.com/RasaHQ/rasa-calm-demo/pull/30/commits/b6ccb2d7ad49db3f081e6776beddc6bbfb5e4526